official/transformer/v2/data_pipeline.py

```diff
@@ -223,10 +227,12 @@ def _read_and_batch_from_files(
 
   # Read files and interleave results. When training, the order of the examples
   # will be non-deterministic.
+  options = tf.data.Options()
+  options.experimental_deterministic = False
```

Comment:
what does this do?

Reply:
It brings back sloppy parallel interleave behavior in V1 dataset.